compiler/test/compilable/extra-files/test23626a.d

```diff
@@ -0,0 +1,49 @@
+template fullyQualifiedName(T...)
+{   
+    enum fullyQualifiedName = !T[0];
+}
+
+void __trace_maybeDumpTupleToFile(Args...)(auto ref const Args args) nothrow @nogc { }
```

Comment:
IDK if it matters, but the name of the function suggests that it should be impure, but a template attribute inference of the will be `pure`.

Reply:
I'd try not to read too much into reductions of 1M+ line code bases. :-)

I could swap this for the further dustmite reduction, making this test even less human-readable.